Sources/5GUIs/Model/InfoDict.swift

```diff
@@ -61,5 +62,6 @@ struct InfoDict: Equatable {
     
     supportedPlatforms = dictionary["CFBundleSupportedPlatforms"] as? [ String ]
                       ?? []
+    JavaX = dictionary["JavaX"] != nil
```

Comment:
What is an example value for `JavaX`?

Reply:
It is a dictionary:
```
		<key>JavaX</key>
		<dict>
			<key>MainClass</key>					<string>org.jd.gui.OsxApp</string>
			<key>JVMVersion</key>					<string>1.8+</string>
			<key>ClassPath</key>					<string>$JAVAROOT/jd-gui-1.6.6-min.jar</string>
			<key>WorkingDirectory</key>				<string>$JAVAROOT</string>
			<key>Properties</key>
			<dict>
				<key>apple.laf.useScreenMenuBar</key>
				<string>true</string>
			</dict>
			<key>VMOptions</key>					<string>-Xms512m</string>
		</dict>
```

Reply:
The app that goes with it is here: [https://github.com/java-decompiler/jd-gui](url)

Reply:
It's an Apple Java SDK application from what I found out.